README.md

```diff
@@ -176,6 +176,16 @@ To check for new samples, use the above script [get_mcm.py](./Production/test/ge
 dasgoclient -query="dataset=/*/RunIISpring16MiniAOD*/MINIAODSIM"
 ```
 
+A convenient config to compute cross sections using the official GenXsecAnalyzer tool is provided, and can be used as follows:
+```
+cmsRun xsecfinder_cfg.py name=Autumn18.TTGamma_Dilept_TuneCP5_13TeV_madgraph_pythia8_ext1 numevents=1e7
```

Comment:
Do you want to mention the `redir` option?

Reply:
Added to README